Uyanda.Coffee.Persistence/Accessors/BeverageAccessor.cs

```diff
@@ -118,7 +118,114 @@ public async Task<BeverageSizeCostModel> UpsertBeverageSizeCostAsync(BeverageSiz
                 return price;
             }
 
-            throw new UpsertSizeCostCommandException();
+            throw new CommandExceptions();
+
+        }
+
+        public async Task<InvoiceModel> UpsertCustomerPurchaseAsync(InvoiceModel invoice)
```

Comment:
This method going down I will not look at, as I have recommended you have only one purchase method

Reply:
Hmm